release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1806,7 +1805,80 @@ With this change, it begins doing so, and conditional updates where the risks ha
+
+* Previously, the Agent-based installer showed the results of secondary checks such as `ping` and `nslookup`, which can harmlessly fail even when the installation succeeds. This could result in errors being displayed despite the cluster installing successfully. With this update, secondary checks only display results if the primary installation checks fail, so that you can use the secondary checks to troubleshoot the failed installation. (link:https://issues.redhat.com/browse/OCPBUGS-8390[*OCPBUGS-8390*])
+
+* Using an IPI `install-config` with the Agent-based Installer results in warning log messages showing the contents of any unused fields. Previously, these warnings printed sensitive information such as passwords. With this update, the warning messages for the credentials fields in the `vsphere` and `baremetal` platform sections have been changed to avoid logging any sensitive information. (link:https://issues.redhat.com/browse/OCPBUGS-8203[*OCPBUGS-8203*])
+
+* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster nodes, and to support SSH traffic from a bastion host to cluster nodes on the same VPC network. Log and debug information can be accurately collected for analysis during installer-provisioned infrastructure failure.(link:https://issues.redhat.com/browse/OCPBUGS-8035[*OCPBUGS-8035*])
+
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973])
+
+* Previously, clusters on Azure Stack Hub could not create new control plane nodes unless the nodes had custom disk sizes, because the default disk size could not be validated. With this update, the default disk size has been set to 128 GB and the installation program enforces user-specified disk size values between 128 and 1023 GB. (link:https://issues.redhat.com/browse/OCPBUGS-6759[*OCPBUGS-6759*])
+
+* Previously, the installation program used port 80 to provide images to the Baseboard Management Controller (BMC) and the deployment agent when installing on bare metal with installer-provisioned infrastructure. This could present security concerns because many types of public traffic use port 80. With this update, the installation program uses port 6180 for this purpose. (link:https://issues.redhat.com/browse/OCPBUGS-8509[*OCPBUGS-8509*])
```

Comment:
```suggestion
* Previously, the installation program used port `80` to provide images to the Baseboard Management Controller (BMC) and the deployment agent when installing on bare metal with installer-provisioned infrastructure. This could present security concerns because many types of public traffic use port `80`. With this update, the installation program uses port `6180` for this purpose. (link:https://issues.redhat.com/browse/OCPBUGS-8509[*OCPBUGS-8509*])
```

Reply:
fixed.